<ui version="4.0" >
 <class>AddCatDialog</class>
 <widget class="QWidget" name="AddCatDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>614</width>
    <height>652</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" >
   <property name="margin" >
    <number>8</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item>
    <widget class="QGroupBox" name="groupBox4" >
     <property name="title" >
      <string>Import File</string>
     </property>
     <layout class="QVBoxLayout" >
      <property name="margin" >
       <number>10</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item>
       <widget class="KUrlRequester" name="DataURL" >
        <property name="toolTip" >
         <string>Enter import data filename</string>
        </property>
        <property name="whatsThis" >
         <string>To import an existing data file, enter its filename here.  You will then describe the contents of the file below.  You can leave the filename blank to construct a catalog file with a valid header, but no data.</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="KListWidget" name="DataFileBox" >
        <property name="enabled" >
         <bool>true</bool>
        </property>
        <property name="font" >
         <font>
          <family></family>
          <pointsize>7</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
          <underline>false</underline>
          <strikeout>false</strikeout>
         </font>
        </property>
        <property name="toolTip" >
         <string>Contents of the imported data file</string>
        </property>
        <property name="whatsThis" >
         <string>When you select a data file in the above line, its contents are displayed here.</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox1" >
     <property name="title" >
      <string>Describe Data Fields</string>
     </property>
     <layout class="QHBoxLayout" >
      <property name="margin" >
       <number>-1</number>
      </property>
      <property name="spacing" >
       <number>0</number>
      </property>
      <item>
       <layout class="QVBoxLayout" >
        <property name="margin" >
         <number>0</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item>
         <widget class="QLabel" name="textLabel1_3" >
          <property name="font" >
           <font>
            <family></family>
            <pointsize>8</pointsize>
            <weight>50</weight>
            <italic>false</italic>
            <bold>false</bold>
            <underline>false</underline>
            <strikeout>false</strikeout>
           </font>
          </property>
          <property name="text" >
           <string>&lt;b>Catalog fields:&lt;/b></string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="DragListBox" name="FieldList" />
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" >
        <property name="margin" >
         <number>0</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item>
         <widget class="QLabel" name="textLabel1_3_2" >
          <property name="font" >
           <font>
            <family></family>
            <pointsize>8</pointsize>
            <weight>50</weight>
            <italic>false</italic>
            <bold>false</bold>
            <underline>false</underline>
            <strikeout>false</strikeout>
           </font>
          </property>
          <property name="text" >
           <string>&lt;b>Available fields:&lt;/b></string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="DragListBox" name="FieldPool" />
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item row="1" column="1" >
      <widget class="KLineEdit" name="Prefix" />
     </item>
     <item row="0" column="1" >
      <widget class="KDoubleNumInput" name="Epoch" >
       <property name="toolTip" >
        <string>The coordinate epoch for the catalog</string>
       </property>
       <property name="value" >
        <double>2000</double>
       </property>
      </widget>
     </item>
     <item row="1" column="0" >
      <widget class="QLabel" name="textLabel4" >
       <property name="text" >
        <string>Catalog name prefix:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2" >
      <layout class="QHBoxLayout" >
       <property name="margin" >
        <number>0</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item>
        <widget class="QLabel" name="textLabel1_4" >
         <property name="text" >
          <string>Symbol color:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="KColorButton" name="ColorButton" >
         <property name="text" >
          <string/>
         </property>
         <property name="color" >
          <color>
           <red>255</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="0" column="0" >
      <widget class="QLabel" name="textLabel3" >
       <property name="text" >
        <string>Coordinate epoch:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2" >
      <layout class="QHBoxLayout" >
       <property name="margin" >
        <number>0</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType" >
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="KPushButton" name="PreviewButton" >
         <property name="enabled" >
          <bool>true</bool>
         </property>
         <property name="text" >
          <string>Preview &amp;Output</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QGridLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item row="1" column="1" >
      <widget class="KUrlRequester" name="CatalogURL" >
       <property name="toolTip" >
        <string>Enter the filename for the output catalog file</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0" >
      <widget class="QLabel" name="textLabel1_2" >
       <property name="text" >
        <string>Catalog name:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0" >
      <widget class="QLabel" name="textLabel1" >
       <property name="text" >
        <string>Save catalog as:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1" >
      <widget class="KLineEdit" name="CatalogName" >
       <property name="toolTip" >
        <string>Enter a name for the catalog</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DragListBox</class>
   <extends>KListWidget</extends>
   <header>widgets/draglistbox.h</header>
   <container>0</container>
   <pixmap></pixmap>
  </customwidget>
  <customwidget>
   <class>KUrlRequester</class>
   <extends></extends>
   <header>kurlrequester.h</header>
   <container>0</container>
   <pixmap></pixmap>
  </customwidget>
  <customwidget>
   <class>KLineEdit</class>
   <extends></extends>
   <header>klineedit.h</header>
   <container>0</container>
   <pixmap></pixmap>
  </customwidget>
  <customwidget>
   <class>KDoubleNumInput</class>
   <extends></extends>
   <header>knuminput.h</header>
   <container>0</container>
   <pixmap></pixmap>
  </customwidget>
  <customwidget>
   <class>KColorButton</class>
   <extends></extends>
   <header>kcolorbutton.h</header>
   <container>0</container>
   <pixmap></pixmap>
  </customwidget>
  <customwidget>
   <class>KListWidget</class>
   <extends></extends>
   <header>klistwidget.h</header>
   <container>0</container>
   <pixmap></pixmap>
  </customwidget>
  <customwidget>
   <class>KPushButton</class>
   <extends></extends>
   <header>kpushbutton.h</header>
   <container>0</container>
   <pixmap></pixmap>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>DataURL</tabstop>
  <tabstop>DataFileBox</tabstop>
  <tabstop>Epoch</tabstop>
  <tabstop>ColorButton</tabstop>
  <tabstop>Prefix</tabstop>
  <tabstop>PreviewButton</tabstop>
  <tabstop>CatalogName</tabstop>
  <tabstop>CatalogURL</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
